src/renderer/shared/api/xcm/xcmService.ts

```diff
@@ -143,94 +181,115 @@ const getRelativeAssetLocation = (
   api: ApiPromise,
   amount: BN,
   assetLocation?: LocalMultiLocation,
-): VersionedMultiAsset | undefined => {
+): Object | undefined => {
   if (!assetLocation) return;
 
   const { parachainId: _, ...location } = assetLocation;
 
-  return api.createType('VersionedMultiAsset', {
-    V2: {
-      id: {
-        Concrete: {
-          parents: 0,
-          interior: Object.values(location).length ? createJunctionFromObject(location) : 'Here',
+  return {
+    V2: [
+      {
+        id: {
+          Concrete: {
+            parents: 0,
+            interior: Object.values(location).length ? createJunctionFromObject(location) : 'Here',
+          },
+        },
```

Comment:
Seems like we can split construction of `id` from the rest of the asset (`fungible` part is always the same)